build/artifacts.props

```diff
@@ -182,6 +182,7 @@
     <PackageArtifact Include="Microsoft.EntityFrameworkCore" Category="ship" />
     <PackageArtifact Include="Microsoft.Extensions.ApiDescription.Design" Category="noship" />
     <PackageArtifact Include="Microsoft.Extensions.ApplicationModelDetection" Category="noship" />
+    <PackageArtifact Include="Microsoft.Extensions.Buffers.Testing.Sources" Category="noship"/>
```

Comment:
Can you also add back the config for Microsoft.Extensions.Buffers.MemoryPool.Sources? This was a new .sources package in 2.2 (found in https://github.com/aspnet/AspNetCore/tree/release/2.2/src/Shared/Buffers.MemoryPool)